client/src/components/LoginSession/LoginSession.js

```diff
@@ -0,0 +1,149 @@
+/* global window */
+import React, { useState } from 'react';
+import confirm from '@silverstripe/reactstrap-confirm';
+import Config from 'lib/Config'; // eslint-disable-line
+import backend from 'lib/Backend';
+import moment from 'moment';
+import i18n from 'i18n';
+import PropTypes from 'prop-types';
+import jQuery from 'jquery';
+import 'regenerator-runtime/runtime';
```

Comment:
Can you clarify what this does?

Reply:
This is necessary for storybook to load successfully.